frb_codegen/src/generator/dart/mod.rs

```diff
@@ -24,65 +24,49 @@ use convert_case::{Case, Casing};
 use log::debug;
 
 use crate::ir::IrType::*;
-use crate::ir::*;
 use crate::method_utils::{FunctionName, MethodNamingUtil};
 use crate::others::*;
-use crate::utils::BlockIndex;
+use crate::{ir::*, Opts};
 
 pub struct Output {
     pub file_prelude: DartBasicCode,
     pub decl_code: DartBasicCode,
     pub impl_code: DartBasicCode,
+    pub needs_freezed: bool,
```

Comment:
Btw I see code quality improved - good job! (Not sure why my previous code review missed a few all bad code taste...)